ros2_generate_interface_docs/ros2_generate_interface_docs/utils.py

```diff
@@ -134,11 +134,11 @@ def generate_index(package, file_directory, timestamp, msg_list=[], srv_list=[],
     :type package: str
     :param file_directory: directory where the index site will be located
     :type file_directory: str
-    :param msg_list: list with msgs for the specific package name
+    :param msg_list: messages for the specific package name
```

Comment:
Nitpick: From the description it could mean "message instances" are being passed. I assume it is message names though, right? How about being specific then with:

```
:param msg_list: name of messages
```

Same for the other parameters.